src/type-layout.md

```diff
@@ -383,12 +369,22 @@ the primitive integer types. That is: `u8`, `u16`, `u32`, `u64`, `u128`,
 
 Primitive representations can only be applied to enumerations and have
 different behavior whether the enum has fields or no fields. It is an error
-for [zero-variant enumerations] to have a primitive representation.
+for [zero-variant enumerations] to have a primitive representation. Combining
+two primitive representations together is an error.
 
-Combining two primitive representations together is unspecified.
+<span id="c-primitive-representation">Combining the `repr(C)`
+and a primitive representation is only defined for enums with fields. The
```

Comment:
This sentence makes it sound like the discussion should be in
```
#### Primitive Representation of Enums With Fields
```
That subsection really has two parts: with and without `repr(C)`.

Reply:
I moved the paragraph to the end of the `Primitive Representation of Enums With Fields`, is it better ?